### ISSUE TITLE
'Disabled item' font color in contextual menu has color contrast ratio of 2.6:1.

### ISSUE THREAD

**Issue by shail15** [microsoft/fluentui#18674]
### Environment Information

- **Package version(s)**: ( "fluentui/react": "^8.17.3",)
- **Browser and OS versions**: (Windows OS: Version Dev (Build 21390.2025)
Microsoft Edge Dev: Version 93.0.916.1 (Official build) dev (64-bit))

### Describe the issue:

The text *'Disabled item'* has a color contrast ratio 2.6:1, which is less than minimum color contrast ratio of 4.5:1.

*Expected Results:*

The text should conform to the standards and have a minimum color contrast ratio of 4.5:1

![image](https://user-images.githubusercontent.com/73106070/123074770-3d326480-d435-11eb-9218-2f7f1472f2fb.png)

*User Impact:*

Users with Vision impairment  will be affected as they will not be able to make out the text if it does not has a minimum contrast ratio to it's background. 

### Please provide a reproduction of the issue in a codepen:

1. Visit https://developer.microsoft.com/en-us/fluentui#/controls/web/contextualmenu
2. Open the Default Contextual Menu
3. Check the color contrast ratio for 'Disabled item' item text color.
4. Tool used : Color Contrast Analyser (CCA)

  

#### Actual behavior:

Color contrast ratio is 2.6:1.

#### Expected behavior:

Color contrast ratio should be 4.5:1

### Documentation describing expected behavior

MAS Link: [ MAS 1.4.3 - Contrast (Minimum)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7ba73546c7-765f-489c-b18f-afb659fe99e6%7d)

**Comment by shail15** [comment]
We confirmed using Accessibility Insights tool, disabled items are an exception and no minimum standards apply for color contrast.
